RHEL6_7/selinux/RestoreConfig/postupgrade.d/restoreConfig.sh

```diff
@@ -2,7 +2,25 @@
 
 SEMANAGE_EXPORT_FILE=semanage_export
 
+semanage_syntax_update() {
+
```

Comment:
I'd also recommend using `local` to prevent defining globals.  (As a side effect, this will allow you to use shorter names.)

Reply:
Another hint: to increase readability and prevent issues caused by later changes, it's good practice to load arguments into variables first.  Ie. don't use `$1` in code, instead do

    local file=$1

and later use `$file`.

Reply:
@AloisMahdal: I prefer to not use the `file` name in the bash as the file corresponds to the `/usr/bin/file` utility. However I don't have anything against renaming.

Reply:
I meant using `local` in general, ie. it wouls not just apply particularly to this `file` variable but also `ftype_opts`, `bool_opts`, `opt`, etc.  Anyway, no need to change it here, myfeedback was meant as style guidance.  There would be actually **many** places where this should be changed.

@pirat89 No, one should use concise and short names.  /usr/bin/file is irrelevant here.  Naming *variable* same as *some completely irrelevant file* is not a conflict.